index.html

```diff
@@ -0,0 +1,274 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <link
+      rel="stylesheet"
+      href="https://use.fontawesome.com/releases/v5.15.4/css/all.css"
+    />
+    <link rel="stylesheet" href="style.css" />
+    <title>Kelwilson Dynamic Tech World</title>
+  </head>
+  <body>
+    <header class="header">
+      <div id="menu" class="fas fa-bars menu-icon"></div>
+      <div class="navbar-nav">
+        <ul>
+          <li>
+            <i class="fab fa-github"></i>
+          </li>
+          <li>
+            <i class="fab fa-twitter"></i>
+          </li>
+          <li>
+            <a href="#">english</a>
+          </li>
+          <li>
+            <a href="#">myPage</a>
+          </li>
+          <li>
+            <a href="#">logOut</a>
+          </li>
+        </ul>
+      </div>
+      <nav class="navbar">
+        <a href="index.html" class="logo">
+          <img src="./images/kelwilson-logo-removebg-preview.png" alt="logo" />
+        </a>
+        <ul>
+          <li>
+            <a href="about.html" id="about-link">About</a>
+          </li>
+          <li>
+            <a href="#catalog">Catalog</a>
+          </li>
+          <li>
+            <a href="#testimonial">Testimonial</a>
+          </li>
+          <li>
+            <a href="#courses"> Course-Catalog</a>
+          </li>
+          <li><a href="#" class="btn" id="start-btn">Start</a></li>
+        </ul>
+      </nav>
```

Comment:
- [ ] Kindly adjust the margins and the size of the fonts of the navbar section for the `Homepage and About page` to match that of the design template as much as possible
Your design
![Capture](https://user-images.githubusercontent.com/85095702/175772872-0b2b0391-52c4-4f4f-a1c9-a1a394f5dd83.jpg)
Behance template
![figma](https://user-images.githubusercontent.com/85095702/175772910-6f8ea719-fe0c-44e0-bfb7-8c9713e6089c.jpg)

- [ ] Also adjust the font size of the nav links in the mobile section, and also the size of the logo
![Capture5](https://user-images.githubusercontent.com/85095702/175773233-a568e3c1-2f35-47c8-bc50-337685c4253d.jpg)